tests/unit/testResources/schemaRegistry.ts

```diff
@@ -1,7 +1,12 @@
-import { CCloudSchemaRegistry } from "../../../src/models/schemaRegistry";
+import { CCloudSchemaRegistry, LocalSchemaRegistry } from "../../../src/models/schemaRegistry";
 import { TEST_CCLOUD_ENVIRONMENT, TEST_CCLOUD_PROVIDER, TEST_CCLOUD_REGION } from "./environments";
 
-export const TEST_SCHEMA_REGISTRY: CCloudSchemaRegistry = CCloudSchemaRegistry.create({
+export const TEST_LOCAL_SCHEMA_REGISTRY: LocalSchemaRegistry = LocalSchemaRegistry.create({
```

Comment:
Currently only used in https://github.com/confluentinc/vscode/blob/8b304a5e5c11c3fc83ae7b29e4da0e30f5adabb8/src/viewProviders/resources.test.ts#L109-L125